code/parse/sexp.cpp

```diff
@@ -12411,6 +12418,52 @@ void sexp_set_player_orders(int n)
 	}
 }
 
+void sexp_set_order_allowed_for_target(int n)
+{
+
+	auto ship_entry = eval_ship(n);
+
+	if (!ship_entry || !ship_entry->shipp) {
+		return;
+	}
+
+	auto shipp = ship_entry->shipp;
+
+	const std::set<size_t>& default_orders = ship_set_default_orders_against();
+	n = CDR(n);
+	bool allow_order;
+	allow_order = is_sexp_true(n);
```

Comment:
You can do `bool allow_order = is_sexp_true(n)`

Reply:
Done